### ISSUE TITLE
Rage Gain in Cat form

### ISSUE THREAD

**Issue by Styggejoe** [wowsims/wotlk#2081]
**Describe the bug**
Rage bar gains while in cat form, not losing rage when going into bear, inflating Feral dps logs

**To Reproduce**
1. Go to 'Feral druid(cat) sim -> Results -> Timeline'
3. Scroll down to 'Start of fight, see rage gain while in cat, inflating dps'

**Expected behavior**
Rage isn't generated while in catform.

**Sim Links and Screenshots**
If applicable, add links to your sim settings or screenshots to help explain your problem.
Picture of rage gain
![image](https://user-images.githubusercontent.com/120034310/206209448-defb0d14-f1c7-4fcb-a0e9-d61280705f8c.png)

Link to sim
https://wowsims.github.io/wotlk/feral_druid/#eJztU0FIVFEUfff+P+Pzpc0zIl4yI7+/CBHE9//ngwUxf1q5/NCiNoFShlCLYQiahGAKgnSlVIKfyE2LICgpGGpy0NCKWYyIoFEYhERMROQq3Nl7M4VIEEG16yzuve+e+859XO5jByl2gsReCDDEfhiCLOahADAGcBdgGnAJoAZwNEHBwgD6IIS8ZvaQ9rXYFYiH5wYuDuYo5dA+DSzBJ2dQfEzZ8eI8TlWQNfPNNbSN8mNkLfy+4t6lbEMTTbxUQnEzqYItFWymWCtfKaK42mGbqpowxh+o442kDUTJfi2jGE/acUVprQT/9ArFF9VHaS1WtfgjpTKRtI1FRZs8mtF29Im2b19rO1fSTReeoqimGFGJahHtVooSQvOEmYdrAF1jSP5jB4YpBY4CLXgIPUxQ5CBiFhwhxMoEsb6Ok4T0Z7J6G9rYLrr1A8BPCxiF4/IX6Pal50rX8VzPdaUjpe856uxJz5G+43S70lcVUuV+wjoI+mIW+fNZFHfeoxWtYOf8MsqXy1gDcwOaIjjMONtNR/ZZ229icWbSD/cM7fl1tQ7aj1cbfupZ3dPbI8YbIBES+zM210cwlwvaGsPYG+yPJjUq6UONzFL6wJk6VtNuGbO3Lg23hAPnTw1dGMydtVzfOibov16qjcvB35ApVBb+REf2dmUzv1tcH15hYj34fqOWVp/vG5lwpRg=

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jarveson** [comment]
Nope, this is just a quirk of the implementation currently, yes u gain rage in car, but it's correctly removed down to 0 or 10 with furor when shifting into bear
